en/tutorial/getting-started.md

```diff
@@ -5,11 +5,11 @@
 
 First, install ECharts using one of the following methods:
 
-* Download official source release from [Apache ECharts (incubating) website](${websitePath}/download.html). Then [build](https://github.com/apache/incubator-echarts#build) from the source release.
+* Download official source release from [Apache ECharts (incubating) website](${websitePath}/en/download.html). Then [build](https://github.com/apache/incubator-echarts#build) from the source release.
 
 * Download from [GitHub](https://github.com/apache/incubator-echarts/releases)
 
-* Using npm: `npm install echarts --save`. [Using ECharts with webpack](${websitePath}/tutorial.html#Use%20ECharts%20with%20webpack)
+* Using npm: `npm install echarts --save`. [Using ECharts with webpack](tutorial.html#Use%20ECharts%20with%20webpack)

```

Comment:
Would be better: `${websitePath}/en/tutorial.html#Use%20ECharts%20with%20webpack`